packages/appstore/src/components/elements/account-wizard/containers/wallet-selector-form/wallet-selector.tsx

```diff
@@ -13,7 +13,17 @@ import {
 import { isMobile, isDesktop } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 
-const WalletSelector: React.FC = (props: any) => {
+type TProps = {
+    onSubmit: any;
+    getCurrentStep: any;
+    goToNextStep: any;
+    validate: any;
+    onSubmitEnabledChange: any;
+    selected_step_ref: any;
+    value: any;
+};
```

Comment:
```suggestion
type TProps = {
    onSubmit: any;
    getCurrentStep: any;
    goToNextStep: any;
    validate: any;
    onSubmitEnabledChange: any;
    selected_step_ref: any;
    value: any;
};
```
Best to not use `any` and use more specific type declarations. Naming we should use something more readable  eg;`WalletSelectorProps`

Reply:
Done